webvtt.go

```diff
@@ -257,7 +262,27 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 			case webvttBlockNameComment:
 				comments = append(comments, line)
 			case webvttBlockNameStyle:
-				// TODO Do something with the style
+				styleLines = append(styleLines, line)
```

Comment:
I don't really see the point of the logic below but I may be wrong.

If we make `WebVTTStyle` a `[]string`, we can `.WebVTTStyle = append(.WebVTTStyle, line)` instead.

Regarding the style id, I would create a `const webVTTDefaultStyleID = "astisub-webvtt-default-style-id"` and use it as `styleID` below.

That would do something like this:

```go
// Create style
if_, ok := o.Styles[webVTTDefaultStyleID]; !ok {
  o.Styles[webVTTDefaultStyleID] = &Style{
    InlineStyle: &StyleAttributes{},
    ID: webVTTDefaultStyleID,
  }
}

// Append line
o.Styles[webVTTDefaultStyleID].InlineStyle.WebVTTStyle = append(o.Styles[webVTTDefaultStyleID].InlineStyle.WebVTTStyle, line)
```

Reply:
In fact, there can be multiple STYLE tokens in WebVTT according to the example 3 in chapter 1.3 of https://www.w3.org/TR/webvtt1/. That is why I separate the STYLE into multiple items, I don't know whether we could combine them into one without any side effect or not, so I decided to keep all STYLE tokens. Besides, it is more convenient when we merge two WebVTT files which contains different STYLE by this way, the CSS will be identified by its selector, then we can add more CSS into the merged file by comparing the selector instead of just keep the STYLE of the master file. (if your are going to merge file2 into file1, then file1 is the master file)

Reply:
It sounds weird to me merging CSS using `Styles` id 🤔 Even if global style is split among several `STYLE` tags, there still is one and only global style global to all webvtt file, hence I'd rather store it in one pre-determined style id, as I suggested.

If we decide to tackle merging CSS then I would suggest transforming `WebVTTStyle` from `[]string` to `map[string][]string`, with the map key being the CSS selector.

Reply:
You are right, merging CSS is complicated and we should just keep the master file style, then let user know about that so they can aware instead of merging and make thing mess up.